_posts/2017-05-02-newsletter.md

```diff
@@ -0,0 +1,207 @@
+- In addition to the effort from the vim guys (7.4.{2055,2057,2058}), `eval.c` has been refactored and error messages were improved (**@ZyX-I**, https://github.com/neovim/neovim/pull/5119)
+- The `id` function is now available, and `printf(%p)` is finally usefull (**@ZyX-I**, https://github.com/neovim/neovim/pull/6095)
+ - All providers are now disabled if `g:loaded_*` exists (**@justinmk**, https://github.com/neovim/neovim/commit/2f38ed11c98a35b7abe53405d8f5f41cb1054f8f)
+ - `setpos` can now set lowercase marks in other buffers (**@hardenedapple**, https://github.com/neovim/neovim/pull/5753)
+ - You can now check `v:exiting` to see if Neovim is exiting (**@mhinz**, https://github.com/neovim/neovim/pull/5651)
+ 
+Now let's see what's in store for the next release. You can get all of this by installing neovim [from the latest source](https://github.com/neovim/neovim/wiki/Installing-Neovim#install-from-source) today.
+
+### The near future: 0.2.1-dev
+
+#### 0.2.1-dev for Users
+- Windows can now use individual highlighting (**@bfredl**, https://github.com/neovim/neovim/pull/6700)
+- Even more terminal improvements (**@jdebp**, https://github.com/neovim/neovim/pull/6816)
+- The command line can now be colored (**@ZyX-l**, https://github.com/neovim/neovim/pull/6364)
+- You can now configure custom clipboard providers (**@nhooyr**, https://github.com/neovim/neovim/pull/6030)
+- For Windows users:
+  - Neovim can now be used on the console (**@equalsraf**, https://github.com/neovim/neovim/pull/6315)
+  - `:terminal` has been implemented (**@erw7**, https://github.com/neovim/neovim/pull/7007)
+  -  Path handling on windows has been improved (**@aignas**, https://github.com/neovim/neovim/pull/7349)
+- For Linux users:
+  - You can now download AppImage packages (**@AdnoC**, https://github.com/neovim/neovim/pull/6638)
+  - Function keys now work in the terminal UI (**@rjmill**, https://github.com/neovim/neovim/pull/5014)
+  - Very long lines do not crash Neovim anymore (**@Grimy**, https://github.com/neovim/neovim/pull/3527)
+  - `cquit` now takes an optional error code as argument (**@joshleeb**, https://github.com/neovim/neovim/pull/7336)
+  - A newly created `:terminal` will now stay in normal mode (**@nelstrom**, https://github.com/neovim/neovim/pull/6808)
+  - True colors in tmux have been fixed (**@DarkDefender**, https://github.com/neovim/neovim/pull/7100)
+  - Further improvements for vimtutor (**@fmoralesc**, https://github.com/neovim/neovim/pull/7028)
+  - Adjust terminal buffers when using the number column (**@tecywiz121**, https://github.com/neovim/neovim/pull/7440)
+
+
+#### 0.2.1-dev for Developers
+- Neovim sources are now analyzed with PVS (**@ZyX-l**, https://github.com/neovim/neovim/pull/6493)
+- External UIs can now draw the tabline (**@dzhou121**, https://github.com/neovim/neovim/pull/6583)
+- First usage of lua to implement core functionality (**@ZyX-l**, https://github.com/neovim/neovim/pull/6789)
```

Comment:
While not wrong, would it not be informative to describe what it does for developers? (lua files can be required from `lua/` dir in runtime and plugins)

Reply:
Sure, I just did not realize that :)